mode-apis-base/src/entityModels.ts

```diff
@@ -414,6 +418,29 @@ export interface Entity {
         readonly metricsDefinitionId: string;
         readonly metricsId: string;
     }[] | undefined;
+
+    /**
+     * The summary of the entity instance's descendants. It is a map of the descendants' entityClass to the number of instances for that class. e.g.
+     *      entityInstance: {
+     *          entityId: "customer_1"
+     *          entityClass: "customer",
+     *          homeId: 1234,
+     *          ...
+     *          summary: {
+     *              site: 3,
+     *              control_server: 5,
+     *              ruber_3000: 58,
+     *          }
+     *      }
+     *
+     * Base on the data above. the "customer_1" has
+     *      - 3 entity instances of class "site"
+     *      - 5 entity instances of class "control_server"
+     *      - 58 entity instances of class "ruber_3000"
+     */
+    readonly summary?: {
```

Comment:
[TBD] I call this `summary` for now but if you like to use other name for this attribute e.g. `statistics`, let me know and I can change the front end to use the same name.

Reply:
`summary` works for me! `statistics` would be okay, but `summary` makes more sense to me in the expected use case scenarios.

I wonder if it's better `treeSummary` would be more descriptive. (summary/statistics was a little bit too vague I though

Reply:
Maybe have some people vote for the names. Add more names that you think work better.

- `summary`
- `statistics`
- `treeSummary`
- `entitiesSummary`
- `descendantSummary`

Reply:
https://docs.google.com/document/d/1qLt_HquaPOckJuaqK3lxiRqzUG4C2pcmjdiCdQ7wUkc/edit?disco=AAAAo6wS8Qw

I'm writing a design doc. I will ask reviewers to vote.

Reply:
`descendantsSummary` is most accurate for me

Reply:
yeah, I feel so too.

Reply:
Okay, from the discussion in another place, I think `descendantsSummaries` is the most viable name.